src/components/test-slot/test-outcome/__tests__/test-outcome.spec.ts

```diff
@@ -37,6 +37,27 @@ describe('Test Outcome', () => {
     examiner: { staffNumber: '123', individualId: 456 },
   };
 
+  const categoryPages = [
+    { category: TestCategory.B, pageConstant: CAT_B },
+    { category: TestCategory.BE, pageConstant: CAT_BE },
+    { category: TestCategory.C, pageConstant: CAT_C },
+    { category: TestCategory.CE, pageConstant: CAT_C },
+    { category: TestCategory.C1, pageConstant: CAT_C },
+    { category: TestCategory.C1E, pageConstant: CAT_C },
+    { category: TestCategory.D, pageConstant: CAT_D },
+    { category: TestCategory.DE, pageConstant: CAT_D },
+    { category: TestCategory.D1, pageConstant: CAT_D },
+    { category: TestCategory.D1E, pageConstant: CAT_D },
+    { category: TestCategory.EUAM1, pageConstant: CAT_A_MOD1 },
+    { category: TestCategory.EUA2M1, pageConstant: CAT_A_MOD1 },
+    { category: TestCategory.EUAM1, pageConstant: CAT_A_MOD1 },
+    { category: TestCategory.EUAMM1, pageConstant: CAT_A_MOD1 },
+    { category: TestCategory.EUAM2, pageConstant: CAT_A_MOD2 },
+    { category: TestCategory.EUA2M2, pageConstant: CAT_A_MOD2 },
+    { category: TestCategory.EUAM2, pageConstant: CAT_A_MOD2 },
+    { category: TestCategory.EUAMM2, pageConstant: CAT_A_MOD2 },
```

Comment:
The categories in bold below appear to be missing, maybe these categories are not done yet?

EUAM1 = 'EUAM1',
EUA2M1 = 'EUA2M1',
EUAMM1 = 'EUAMM1',
**EUA1M1 = 'EUA1M1',**
EUAM2 = 'EUAM2',
EUA2M2 = 'EUA2M2',
EUAMM2 = 'EUAMM2',
**EUA1M2 = 'EUA1M2',**